component/admin/models/media.php

```diff
@@ -375,8 +387,10 @@ public function deleteFile($id)
             ->where($db->qn('media_id') . ' = ' . $id);
         $db->setQuery($query);
 
-        if (!$db->execute()) {
-            $this->setError($db->getErrorMsg());
+        try {
+            $db->execute();
+        } catch (\RuntimeException $e) {
+            \JFactory::getApplication()->enqueueMessage(\JText::_('COM_REDSHOP_ERROR_FILE_DELETING'), 'error');
```

Comment:
@hoangvan149dhv Same here, please using namespaces, if it takes long code, you can use `Use` keyword at start of php code file. Please do the same with all Class with `J` prefix.